src/main/java/org/folio/circulation/support/ItemRepository.java

```diff
@@ -354,4 +358,49 @@ private ItemRepository(
       .thenComposeAsync(this::fetchMaterialType)
       .thenComposeAsync(this::fetchLoanType);
   }
+
+  public CompletableFuture<Result<ResultItemContext>> getAllItemsByField(String fieldName, String fieldValue) {
```

Comment:
These methods appear to be specific to reporting, please move them to a class that better describes that use

Reply:
Moved